doc/source/admin/galaxy_options.rst

```diff
@@ -1025,6 +1025,40 @@
 :Type: str
 
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``mulled_resolution_cache_type``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    Mulled resolution caching. Mulled resolution uses external APIs of
+    quay.io, these requests are caching using this and the following
+    parameters
+:Default: ``file``
+:Type: str
+
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``mulled_resolution_cache_data_dir``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    Data directory used by beaker for caching mulled resolution
+    requests.
+:Default: ``citations/data``
```

Comment:
See my comments on https://github.com/galaxyproject/galaxy/pull/8741 from today.

Reply:
I'll rebase over that when it's fixed in dev.  I don't know what the defaults should be or I'd do it now -- just `mulled/data` `mulled/locks` ?

Reply:
Neither do I! It was just a heads up.